src/shogun/distributions/classical/GaussianDistributionNew.cpp

```diff
@@ -0,0 +1,244 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Written (W) 2013 Heiko Strathmann
```

Comment:
Could you put a BSD license here and add your name?
See any recent Shogun file for a template